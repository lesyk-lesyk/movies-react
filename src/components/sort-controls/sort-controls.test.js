import React from 'react';
import { SortControls } from './index';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

jest.mock('../sort-item/sort-item');

describe('SortControls', () => {
  it('renders', () => {
    const callback = () => { };
    const tree = renderer
      .create(<SortControls />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
