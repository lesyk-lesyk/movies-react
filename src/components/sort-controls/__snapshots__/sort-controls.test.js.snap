// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortControls renders 1`] = `
<div
  className="sort-controls"
>
  Sort by
  <SortItem
    onClick={[Function]}
    sortBy="release_date"
  >
    release date
  </SortItem>
  <SortItem
    onClick={[Function]}
    sortBy="vote_average"
  >
    rating
  </SortItem>
</div>
`;
